docs/collected-metrics/logins/metrics.md

```diff
@@ -0,0 +1,31 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
```

Comment:
The associated `metrics.yaml` file for logins predates this PR and therefore shouldn't generate any changes from what exists. However, the top-level name has been changed and the metric names are from the YAML file for places, which I created in this PR.